debian/postinst

```diff
@@ -115,10 +116,28 @@ EOF
     fi
 }
 
+
+mark_reboot_for_fips_pro() {
```

Comment:
Maybe I am missing something here but this function doesn't seem to called in the postinst script

Reply:
just added it to the defaul call patch in configure)